modules/cnv-creating-local-block-pv.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes-block.adoc
+// * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume-block.adoc
+// * cnv/cnv_users_guide/cnv-uploading-local-disk-images-block.adoc
+
+[id="cnv-creating-local-block-pv_{context}"]
+= Creating a local block PersistentVolume
+
+Create a local block PersistentVolume (PV) on a node by populating a file and 
+mounting it as a loop device. This loop device can then be referenced in a 
+PV configuration as a `Block` volume and used as a block device for a 
+virtual machine image. 
+
+.Procedure
+
+. Log in as `root` to the node on which to create the local PV. This procedure 
+uses `node01` for its examples.
+
+. Create a file and populate it with null characters. The following example creates a file
```

Comment:
Updated to read: "Create a file and populate it with null characters so that it can be used as a block device.
The following example creates a file `loop10` with a size of 2Gb (20 100Mb blocks)."?